packages/puppeteer-core/src/cdp/Input.ts

```diff
@@ -552,57 +553,103 @@ export class CdpMouse extends Mouse {
 /**
  * @internal
  */
-export class CdpTouchscreen extends Touchscreen {
+class CdpTouch implements TouchHandle {
+  #started = false;
+  #touchScreen: CdpTouchscreen;
+  #touchPoint: Protocol.Input.TouchPoint;
   #client: CDPSession;
   #keyboard: CdpKeyboard;
 
-  constructor(client: CDPSession, keyboard: CdpKeyboard) {
-    super();
+  constructor(
+    client: CDPSession,
+    touchScreen: CdpTouchscreen,
+    keyboard: CdpKeyboard,
+    public id: number,
+    touchPoint: Protocol.Input.TouchPoint,
+  ) {
     this.#client = client;
+    this.#touchScreen = touchScreen;
     this.#keyboard = keyboard;
+    this.#touchPoint = touchPoint;
   }
 
   updateClient(client: CDPSession): void {
     this.#client = client;
   }
 
-  override async touchStart(x: number, y: number): Promise<void> {
+  async start(): Promise<void> {
+    if (this.#started) {
+      return;
```

Comment:
Let's throw an error here, this should no be reachable, and it better to fix issues if we encounter them then fail them silently.

Reply:
We could throw an error, but how do I test that? `start()` is not part of the public api after all

Reply:
I added the error.